jdaviz/configs/imviz/wcs_utils.py

```diff
@@ -383,7 +383,7 @@ def _prepare_rotated_nddata(real_image_shape, wcs, rotation_angle, refdata_shape
     # create a fake NDData (we use arange so data boundaries show up in Imviz
     # if it ever is accidentally exposed) with the rotated GWCS:
     ndd = NDData(
-        data=np.arange(sum(refdata_shape), dtype=np.int8).reshape(refdata_shape),
+        data=np.arange(np.prod(refdata_shape), dtype=np.int8).reshape(refdata_shape),

```

Comment:
Hmm does plain `sum` not work for you? The shape should be a plain tuple if ints.

Reply:
If shape is (10, 10), sum is 20. We want the product (100).

Reply:
Oh, right... I didn't catch that with `(2, 2)`. Doh!